### DISCUSSION TITLE
Champ "Email de gestionnaire non notifié" sur admin

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1650] [Idées]
**Besoin :**
Suite à un import massif, j'ai besoin d'ajouter l'adresse électronique d'un gestionnaire en mode "non notifié" à une cantine pour laquelle les référentes restau co en Département n'avaient pas renseigné d'adresse dans le fichier d'import. Mais je ne trouve pas de champ correspondant sur la fiche de la cantine sur notre back office.
=> Créer un tel champ.

**Intérêt :**
Éviter de vous solliciter, les développeurs, pour ce genre de demandes. Mais peut-être que ça ne vaut pas le coup d'investir du temps de développement pour une demande qui restera vraisemblablement ponctuelle.

Qu'en pensez-vous, @JenniferStephan, @alemangui et @hfroot?

**Comment by alemangui** [comment]
Hello, on a vu avec JB, mais pour celles et ceux qui trouveront cette conversation, voici les instructions pour rajouter un email d'un gestionnaire (qui n'a pas encore un compte chez nous) à une cantine sans lui envoyer un email : 
- D'abord ça vaut la peine de vérifier que l'email n'est pas déjà assigné à un compte. Vous pouvez chercher [dans l'admin](https://ma-cantine.beta.gouv.fr/admin/data/user/) par email et vous assurer qu'aucun compte n’apparaît. 
- Maintenant, ajoutons une invitation de gestion pour l'email en question. Ça peut se faire dans l'apparté [invitations de gestion](https://ma-cantine.beta.gouv.fr/admin/data/managerinvitation/) dans l'admin, en cliquant en haut à droit sur _Ajouter invitation de gestion_
- Renseignez l'email et sélectionnez la cantine pour laquelle cette invitation doit être créée

Aucun email ne sera envoyé, mais lors de la création de compte de l'utilisateur ayant cet email, la cantine sera automatiquement liée et l'utilisateur se trouvera gestionnaire de la cantine.

**Reply by ghost** [reply]
Merci encore pour l'explication, @alemangui :-)
